blockchain/state_manager/src/lib.rs

```diff
@@ -1073,10 +1069,7 @@ where
 
     /// Return the heaviest tipset's balance from self.db for a given address
     pub fn get_heaviest_balance(&self, addr: &Address) -> Result<TokenAmount, Error> {
-        let ts = self
-            .cs
-            .heaviest_tipset()
-            .ok_or_else(|| Error::Other("could not get bs heaviest ts".to_owned()))?;
+        let ts = self.cs.heaviest_tipset();
         let cid = ts.parent_state();
```

Comment:
```suggestion
        let cid = self.cs.heaviest_tipset().parent_state();
```

Reply:
Rust doesn't like it, asks it be the existing way

Reply:
Then use:
`let cid = *self.cs.heaviest_tipset().parent_state();`

Reply:
Ok